tests/dataset_builders/pie/test_cdcp.py

```diff
@@ -0,0 +1,431 @@
+import dataclasses
+from typing import List
+
+import pytest
+from datasets import disable_caching, load_dataset
+from pytorch_ie import DatasetDict, tokenize_document
+from pytorch_ie.annotations import LabeledSpan
+from pytorch_ie.core import AnnotationList, Document, annotation_field
+from pytorch_ie.documents import (
+    TextBasedDocument,
+    TextDocumentWithLabeledSpansAndBinaryRelations,
+)
+from transformers import AutoTokenizer, PreTrainedTokenizer
+
+from dataset_builders.pie.cdcp.cdcp import (
+    CDCP,
+    CDCPDocument,
+    convert_to_text_document_with_labeled_spans_and_binary_relations,
+    document_to_example,
+    example_to_document,
+)
+from src.document.types import TokenDocumentWithLabeledSpansAndBinaryRelations

```

Comment:
Since this repo will produce a real python package (see https://pypi.org/project/pie-datasets/), we should never import sth from `src`. It should be:
```suggestion
from pie_datasets.document.types import TokenDocumentWithLabeledSpansAndBinaryRelations
```

Accordingly, you need to move the `document` folder into the `pie_datasets` folder. All code should be in `src/pie_datasets` except the dataset scripts (i.e. the `dataset_builders` folder). As defined [here](https://github.com/ArneBinder/pie-datasets/blob/main/pyproject.toml#L13-L15), the final package (named `pie_datasets`) will be build from the that folder.

Reply:
fix in ec6b346